client/dashboard/sites/logs/test/index.test.tsx

```diff
@@ -199,9 +199,6 @@ describe( 'SiteLogs page', () => {
 			const initialLogType = logTypeName !== LogType.PHP ? LogType.PHP : LogType.SERVER;
 			render( <SiteLogs logType={ initialLogType } /> );
 
-			// Wait for data and TabPanel to render
-			await waitFor( () => expect( nock.isDone() ).toBe( true ) );
```

Comment:
Removing these network checks to get the tests passing. The new `initialData` means the network call is no longer needed.

But we shouldn't need to wait explicitly for network calls to finish. We're already using `findBy*` queries, which have a wait built in: https://testing-library.com/docs/dom-testing-library/api-async#findby-queries

Waiting for UI elements that the user would see is better.